lib/request_log_analyzer/version.rb

```diff
@@ -1,3 +1,3 @@
 module RequestLogAnalyzer
-  VERSION = '1.13.4'
+  VERSION = '1.13.5'
```

Comment:
Please don't update the VERSION constant - this is done as part of the release process.